scaffold/convert/create.template.js

```diff
@@ -202,6 +202,98 @@ const getInputFields = (z, bundle) => {
   return responsePromise
     .then((response) => z.JSON.parse(response.content));
 };
+<% }
+
+if (outputFieldFullScripting) {%>
+const getOutputFields = (z, bundle) => {
+  const scripting = require('../scripting');
+  const legacyScriptingRunner = require('zapier-platform-legacy-scripting-runner')(scripting);
+
+  bundle._legacyUrl = '<%= CUSTOM_FIELDS_RESULT_URL %>';
+  bundle._legacyUrl = legacyScriptingRunner.replaceVars(bundle._legacyUrl, bundle, {});
```

Comment:
Since we're creating a "local" `replaceVars`, I think it makes sense to use that here (and making it consistent with input fields).

This happens in a few places below as well.

Reply:
Fixed in 7a5d22f. And since we're using `replaceVars()` in several places, I think it's a good idea to separate the function to a shared utility module.